go/server.go

```diff
@@ -206,6 +208,7 @@ func transactions(c *gin.Context) {
 	})
 }
 
+// This functionality is only relevant for the UK Payment Initiation product.
 // Retrieve Payment for a specified Payment ID
 // https://plaid.com/docs/#payment-initiation
```

Comment:
Can we remove?

Reply:
https://github.com/plaid/quickstart/pull/94/commits/e5a3cdfd703f17d525a395e01585a027aa8acf5a